gateway/modules/eventing/operations_test.go

```diff
@@ -65,3 +71,209 @@ func TestModule_SetRealtimeTriggers(t *testing.T) {
 		})
 	}
 }
+
+func TestModule_IsEnabled(t *testing.T) {
+	tests := []struct {
+		name string
+		m    *Module
+		want bool
+	}{
+		{
+			name: "config is nil",
+			m:    &Module{},
+			want: false,
+		},
+		{
+			name: "config is not nil but enabled is nil",
+			m:    &Module{config: &config.Eventing{}},
+			want: false,
+		},
+		{
+			name: "enabled is true",
+			m:    &Module{config: &config.Eventing{Enabled: true}},
+			want: true,
+		},
+		{
+			name: "enabled is false",
+			m:    &Module{config: &config.Eventing{Enabled: false}},
+			want: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := tt.m.IsEnabled(); got != tt.want {
+				t.Errorf("Module.IsEnabled() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestModule_SendEventResponse(t *testing.T) {
+
+	type args struct {
+		batchID string
+		payload interface{}
+	}
+	tests := []struct {
+		name string
+		m    *Module
+		args args
+	}{
+		{
+			name: "key not present in eventChanMap",
+			m:    &Module{},
+			args: args{batchID: "notbatchid", payload: "payload"},
+		},
+		{
+			name: "event response is sent",
+			m:    &Module{},
+			args: args{batchID: "batchid", payload: "payload"},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+
+			tt.m.eventChanMap.Store("batchid", eventResponse{time: time.Now(), response: make(chan interface{}, 1)})
```

Comment:
You'll have to check if the message is actually received by the channel. Make a listener.